src/bun.js/bindings/InternalModuleRegistry.cpp

```diff
@@ -104,10 +104,10 @@ JSValue initializeInternalModuleFromDisk(
     JSGlobalObject* globalObject,
     VM& vm,
     const WTF::String& moduleName,
-    WTF::String fileBase,
+    const WTF::String& fileBase,
     const WTF::String& urlString)
 {
-    WTF::String file = makeString(ASCIILiteral::fromLiteralUnsafe(BUN_DYNAMIC_JS_LOAD_PATH), "/"_s, WTFMove(fileBase));
+    WTF::String file = makeString(ASCIILiteral::fromLiteralUnsafe(BUN_DYNAMIC_JS_LOAD_PATH), "/"_s, fileBase);
```

Comment:
I would revert this change. Unless I'm missing something: https://borjaportugal.com/2023/12/20/move-semantics/

Reply:
TL;DR: If the user passed an rvalue into `initializeInternalModuleFromDisk`, with your diff applied, a copy would always be created.

Previously, the compiler had the liberty to elide the extra copy.